cardano-node/src/Cardano/Node/Tracing/Tracers/Startup.hs

```diff
@@ -135,12 +137,31 @@ namesStartupInfo = \case
   NetworkConfigLegacy {}                    -> ["NetworkConfigLegacy"]
   P2PWarning {}                             -> ["P2PWarning"]
   P2PWarningDevelopementNetworkProtocols {} -> ["P2PWarningDevelopementNetworkProtocols"]
-  WarningDevelopmentNetworkProtocols {}     -> ["WarningDevelopmentNetworkProtocols"]
+  WarningDevelopmentNodeToNodeVersions {}   -> ["WarningDevelopmentNodeToNodeVersions"]
+  WarningDevelopmentNodeToClientVersions {} -> ["WarningDevelopmentNodeToClientVersions"]
   BICommon {}                               -> ["Common"]
   BIShelley {}                              -> ["ShelleyBased"]
   BIByron {}                                -> ["Byron"]
   BINetwork {}                              -> ["Network"]
 
+
+data VersionTuple a b = VersionTuple a b
+
+-- TODO: provide JSON instance for `BlockNodeToClientVersion`
+instance Show blkVersion => ToJSON (VersionTuple NodeToClientVersion blkVersion) where
+    toJSON (VersionTuple nodeToClientVersion blockVersion) =
+      Aeson.object [ "nodeToClientVersion" .= toJSON nodeToClientVersion
+                   , "blockVersion" .= String (pack . show $ blockVersion)
```

Comment:
@dnadales I left the consensus part for you guys to finish.

Reply:
Are these TODOs relevant?

Reply:
Yes, they are left for the consensus team. I captured this in [the issue](https://github.com/input-output-hk/cardano-node/issues/5000).